packages/@react-spectrum/breadcrumbs/docs/Breadcrumbs.mdx

```diff
@@ -188,3 +151,33 @@ This can be used to maintain layout continuity.
   <Item key="march 2020 assets">March 2020 Assets</Item>
 </Breadcrumbs>
 ```
+
+### Visible items (overflow behavior)
+[View guidelines](https://spectrum.adobe.com/page/breadcrumbs/#Truncation-and-overflow)
+
+Breadcrumbs collapse into a menu by default, showing only a maximum of 4 visible items. This count includes the root
+item and the Breadcrumb menu button if either is visible. If the root item cannot be rendered in the available horizontal
+space, it will be collapsed into the menu regardless of `showRoot`. Note that the last breadcrumb item will automatically
+truncate with an ellipsis instead of collapsing into the menu.
```

Comment:
Should we explain that the menu shows all items with the current page being selected?

Reply:
Yeah, so I tried to be selective with what I mentioned here since the menu containing all items + current page being selected stuff is detailed here: https://spectrum.adobe.com/page/breadcrumbs/#Overflow and I wanted as little overlap as possible. I go into a bit of detail here about the truncation behavior specifically here since we have a `showRoot` prop that doesn't prevent the root from collapsing into the menu which might be confusing for an end user.

Reply:
Breadcrumb's overflow behavior collapses items into a menu when space is limited. Breadcrumbs will only show a maximum of 4 visible items including the root and Breadcrumb menu button, if either are visible.